### PR TITLE
refactor: move loading individual elements to service

### PR DESCRIPTION
**Pull request by andreasphil**
RISDEV-3988

### CODE REVIEW

File: backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/ElementServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+  @Nested
+  class loadElementFromNorm {
+    @Test
+    void itLoadsAnElementFromANorm() {
+      // Given
+      var eli = "eli/bund/bgbl-1/2000/s1/1970-01-01/1/deu/regelungstext-1";
+      var eid = "meta-1";
+
+      var normXml =
```
Comment (hannes-mk) [inline backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/ElementServiceTest.java:29]: How about using a fixture file (just for reasons of readability)?

File: backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/ElementServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                        />
+                      </akn:FRBRExpression>
+                    </akn:identification>
+                  </akn:meta>
+                </akn:act>
+              </akn:akomaNtoso>
+              """;
+
+      var norm = new Norm(XmlMapper.toDocument(normXml));
+      when(loadNormPort.loadNorm(new LoadNormPort.Command(eli))).thenReturn(Optional.of(norm));
```
Comment (hannes-mk) [inline backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/ElementServiceTest.java:54]: Do we know why it's a "Command" here and a "Query" in the use cases?

File: backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/ElementServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+      // When
+      var element = service.loadElementFromNorm(new LoadElementFromNormUseCase.Query(eli, eid));
+
+      // Then
+      assertThat(element).isPresent();
+      assertThat(NodeParser.getValueFromExpression("./@eId", element.get())).contains(eid);
+    }
+
+    @Test
+    void itReturnsEmptyIfNoNormIsFound() {
```
Comment (hannes-mk) [inline backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/ElementServiceTest.java:65]: Maybe "EmptyOptional" to reduce ambiguite of "empty"?

File: backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/ElementServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+      // When
+      var element = service.loadElementFromNorm(new LoadElementFromNormUseCase.Query(eli, eid));
+
+      // Then
+      assertThat(element).isEmpty();
+    }
+
+    @Test
+    void itReturnsEmptyIfNoElementIsFound() {
```
Comment (hannes-mk) [inline backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/ElementServiceTest.java:80]: Maybe: "...EmptyOptional" (cf. above)

File: backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/ElementServiceTest.java [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+      // Given
+      var eli = "eli/bund/bgbl-1/2000/s1/1970-01-01/1/deu/regelungstext-1";
+      var eid = "meta-1000";
+
+      var normXml =
```
Comment (hannes-mk) [inline backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/ElementServiceTest.java:85]: Read norm from disk?

**Review by hannes-mk — COMMENTED** [review]
Looks good to me. Just a handful of ideas.

File: backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/ElementServiceTest.java (hunk 1 above)
Comment (andreasphil) [reply backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/ElementServiceTest.java:29]: IMO this is small enough not to use a fixture

File: backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/ElementServiceTest.java (hunk 5 above)
Comment (andreasphil) [reply backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/ElementServiceTest.java:85]: IMO this is small enough not to use a fixture

File: backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/ElementServiceTest.java (hunk 2 above)
Comment (andreasphil) [reply backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/ElementServiceTest.java:54]: I guess it's related to [Command/Query Separation](https://martinfowler.com/bliki/CommandQuerySeparation.html).

**Pull request merged**